e2e/integration/journey.test.js

```diff
@@ -122,14 +125,22 @@ it('requires employee selection', () => {
   cy.findByRole('button', { name: /submit/i }).click()
   cy.findByText(/form is incomplete/i).should('exist')
   cy.findByRole('combobox', { name: /who is your employer/i }).type(
-    'Cap{enter}'
+    'Capgemini{enter}'
```

Comment:
:x: This is what is hiding all of your problems. The employer list is **not** populated, which is why without making this change it fails:
```
[e2e]   1) requires employee selection:
[e2e] 
[e2e]       AssertionError: expected { Object (firstName, lastName, ...) } to have property 'employer' of 'Capgemini', but got 'Cap'
[e2e]       + expected - actual
[e2e] 
[e2e]       -'Cap'
[e2e]       +'Capgemini'
```
With no items in the list the employer can't be auto-completed, so it's submitted as a _new_ employer. 

:question: Maybe we should explicitly check that the corresponding message (_"This employer will be added to our list. Make sure you typed it correctly."_) isn't shown at this point?

```suggestion
    'Cap{enter}'
```

Reply:
I put 'Cap{enter}' back and the tests are passing. But this message ("This employer will be added to our list. Make sure you typed it correctly.") shows whether the employer exists or is newly typed.

Reply:
That message _isn't_ shown if the employer already exists - the fact that it _was_ being shown is what gave away that `Capgemini` was being entered as a manual addition **not** actually being looked up from the employers data (which wasn't loading correctly due to the response shape mismatches). But it's not critical to add this assertion.